DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Commands/INoteCommands.cs

```diff
@@ -6,6 +6,6 @@ namespace GitHubExtension.Notes.WebApi.Commands
 {
     public interface INoteCommands
     {
-        Task AddNote(Note noteEntity);
+        Task AddOrUpdateNote(Note noteEntity);
```

Comment:
Explain please why you are decided change method name AddNote -> AddOrUpdateNote? What about SOLID?